### PR TITLE
adding ui for viewing errors as admin

### PR DESCRIPTION
**Pull request by mekarpeles**
Unblocking @rchrd2 on accessing webpy error reports

### CODE REVIEW

**Comment by rchrd2** [comment]
Is there security here?

**Comment by mekarpeles** [comment]
Yes -- e.g. https://dev.openlibrary.org/admin/logs?name=2017-04-04/212301317713
One must be an admin to access /admin endpoints

File: openlibrary/plugins/admin/code.py [hunk 1]
```diff
@@ -639,6 +639,16 @@ class imports_by_date:
     def GET(self, date):
         return render_template("admin/imports_by_date", imports.Stats(), date)
 
+class show_log:
+    def GET(self):
+        i = web.input(name='')
+        logname = i.name
+        filepath = config.get('errorlog', 'errors') + '/'+ logname + '.html'
+        print(filepath)
```
Comment (bfalling) [inline openlibrary/plugins/admin/code.py]: Is this `print()` meant to be left in?

**Comment by bfalling** [comment]
LGTM. Merge away.

**Pull request merged**